packages/core/src/v2/constructable.ts

```diff
@@ -14,9 +14,9 @@ export interface Constructable<
   _serviceName: string;
   _entityName: string;
   _defaultServicePath: string;
-  _allFields: Selectable<EntityT>[];
-  _keyFields: Selectable<EntityT>[];
-  _keys: { [keys: string]: Selectable<EntityT> };
+  _allFields: (Field<EntityT> | Link<EntityT>)[];
```

Comment:
This seems like a quick fix to some type issue?

Reply:
No, actually I noticed that along the way. The type before was just wrong.